dlp/api/DlpSample/InspectTemplates.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Google.Api.Gax;
+using Google.Cloud.Dlp.V2;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace GoogleCloudSamples
+{
+    class InspectTemplates
+    {
+        // [START dlp_create_inspect_template]
+        public static string CreateInspectTemplate(
+            string projectId, 
+            string displayName,
+            string description,
+            int likelihood, 
+            int maxFindings,
+            bool includeQuote,
+            string infoTypes)
```

Comment:
This never appears to get used. You can parse them into an actual List by extending DlpSampleBase and calling ParseInfoTypes(infoTypes)

This also tells me that your unit test doesn't check the infoTypes param on the call.